crates/primitives/src/block.rs

```diff
@@ -83,6 +111,50 @@ impl Deref for SealedBlock {
     }
 }
 
+impl DerefMut for SealedBlock {
+    fn deref_mut(&mut self) -> &mut Self::Target {
+        &mut self.header
+    }
+}
```

Comment:
this seems dangerous. because this allows mutating the header when it's supposed to be sealed.

Reply:
Moved them behind cfg (`test`, `test-utils`)